azure-pipelines.yml

```diff
@@ -112,7 +112,7 @@ jobs:
   - task: UsePythonVersion@0
     displayName: 'Use Python 3'
```

Comment:
This `displayName` is not consistent with that of `VerifyWindowsRequirements`.

Reply:
Fixed